docs/Maintainers.md

```diff
@@ -0,0 +1,95 @@
+Community Maintenance Committee
+===============================
+
+This document describes the Community Maintenance Committee of Open Enclave. By
+our liberal contribution policy outlined in our
+[governance model](GovernanceModel.md), maintainers are committers that are
+trusted to grant new committer rights, and grant new membership into the
+Committee.
+
+When making decisions, the Community Maintenance Committee uses a "consensus
+seeking" process. This means that most decisions should be reached by consensus,
+but when that fails, the Committee calls for a vote where the super majority
+(two-thirds) wins. This is to prevent obstructionism by removing the possibility
+of a one person veto.
+
+Committee Members
+-----------------
+
+| Name                 | Company   | Email                         | GitHub Alias   |
+|----------------------|-----------|-------------------------------|----------------|
+| Akash Gupta          | Microsoft | akagup@microsoft.com          | gupta-ak       |
+| Amaury Chamayou      | Microsoft | amaury.chamayou@microsoft.com | achamayou      |
+| Andrew Schwartzmeyer | Microsoft | andschwa@microsoft.com        | andschwa       |
+| Dave Thaler          | Microsoft | dthaler@microsoft.com         | dthaler        |
+| John Kordich         | Microsoft | johnkord@microsoft.com        | johnkord       |
+| Mike Brasher         | Microsoft | mikbras@microsoft.com         | mikbras        |
+| Simon Leet           | Microsoft | simon.leet@microsoft.com      | CodeMonkeyLeet |
+
+Committee Responsibilities
+--------------------------
+
+The primary responsibility of the Committee is to grant new committer rights
+(that is, write access to the main Open Enclave SDK repository or related
+repositories), and to grant new membership into the committee. Conversely, the
+Committee must also remove committer rights and membership from those found to
+be violating the project's Code of Conduct or otherwise negatively affecting the
+project's community health.
+
+This Committee is not intended to make every technical decision, as those should
+generally be made by agreement among committers as PRs are reviewed and merged.
+Where disagreements take place and need further resolution, those can be brought
+up with the Maintainers Committee as part of its responsibility to maintain the
```

Comment:
Lines 42 and 55 mention "Maintainers Committee".  What's the difference between that and the "Community Maintenance Committee"?

Reply:
It should be "Community Maintenance Committee", good catch.